internal/engine/ingester/diff/parse.go

```diff
@@ -137,31 +138,52 @@ func pyNormalizeName(pkgName string) string {
 }
 
 func goParse(patch string) ([]*pb.Dependency, error) {
```

Comment:
General comment: in future, we may want to consider splitting each of these parsers (and their tests) out into separate files. This file is going to keep growing and growing otherwise.

Reply:
Yeah, now that you mention it the way I see it is number of supported ecosystems == number of parsers, but I agree, let's see how this grows and we'll revisit 👍